docs/hello_nextflow/03_hello_plumbing.md

```diff
@@ -0,0 +1,755 @@
+# Part 3: Hello Plumbing
+
+Most real-world workflows involve more than one step.
+In this training module, you'll learn how to connect processes together in a multi-step workflow.
+
+This will teach you the Nextflow way of achieving the following:
+
+1. Making data flow from one process to the next
+2. Collecting outputs from multiple process calls into a single process call
+3. Passing more than one input through a channel
```

Comment:
We just did that in the previous module, right?

Reply:
Do you mean "Passing more than one input through a channel"? Here it's about having multiple 'tracks' in the input channel, eg the data but also some parameters etc., as a opposed to 'several pieces of the same kind of data'. In general this is a point of confusion for which we should try to develop a controlled vocabulary, because 'multiple inputs' ends up being ambiguous

Reply:
Yes, I actually realised what you meant further down. This probably needs to be something like 'passing collections of elements to a single task' or something similar.

Reply:
Are you using 'collections of elements' in the same way I'm using 'input tracks in a channel'?

Reply:
fixed this in a previous pass